docs/setup/compute-ntp.md

```diff
@@ -22,7 +22,6 @@ If your Create® 3 and compute board have an internet connection, this is not re
 
 1. Add the following lines after the `pool #.ubuntu.pool.ntp.org iburst maxsources #` block
 
-        server 192.168.186.2 presend 0 minpoll 0 maxpoll 0 iburst  prefer trust
```

Comment:
This makes the raspi consider itself an NTP server - I'm not sure if that was intended, but it was unnecessary for my setup.